data/jmgrd98.json

```
@@ -12,7 +12,15 @@
         "color": "green",
         "description": "Started my programming studies.",
         "url": "https://www.github.com/jmgrd98/"
-    }
+    },
+    {
+        "title": "Started in open source!",
+        "date": "January 2021",
+        "icon": "FaDollarSign",
+        "color": "green",
+        "description": "Started contributing to open source projects as a way to improve my coding skills.",
+        "url": "https://www.github.com/jmgrd98/"
+    },
```

```suggestion
    }
```
Please remove this comma